client/lib/posts/post-edit-store.js

```diff
@@ -96,6 +98,22 @@ function getCategoryIds( post ) {
 	} );
 }
 
+function getTermIds( post ) {
+	if ( ! post || ! post.terms ) {
+		return;
+	}
+
+	// Skip "default" taxonomies
+	const taxonomies = omit( post.terms, [ 'post_tag', 'category' ] );
```

Comment:
Ha - yes I should have explained that a bit more.  Currently both `category` and `post_tag` components in the editor sidebar update their perspective attributes on the post object, and do not update the `terms` object.  Since this logic here pulls the term ids from `terms`, and the order of the term logic in the API, if we do not filter out `post_tag` and `category` changes are not saved properly.

Longer-term I can see us moving away entirely from the existing token-field and category-selector implementations at which time we can remove this omit statement and all the data can live in `terms`.